packages/amplify-category-auth/src/index.js

```diff
@@ -68,8 +68,10 @@ async function add(context, skipNextSteps = false) {
 async function transformCategoryStack(context, resource) {
   if (resource.service === AmplifySupportedService.COGNITO) {
     if (canResourceBeTransformed(resource.resourceName)) {
-      await generateAuthStackTemplate(context, resource.resourceName);
+      generateAuthStackTemplate(context, resource.resourceName);
     }
+  } else if (resource.service === 'S3') {
```

Comment:
Why is  S3 referenced in Auth ? Is this downstream service specific ?

Reply:
I'm not sure - this is a revert, so the https://github.com/aws-amplify/amplify-cli/pull/9847 PR removes this.